docs/runtimes/application.ipynb

```diff
@@ -92,6 +92,74 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import mlrun.runtimes.application
</pre><p>this import should also work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
actually, this is better:
```
import mlrun.runtimes

# and user:
mlrun.runtimes.ApplicationRuntime
```